tests/integration/test_job.py

```diff
@@ -104,8 +105,9 @@ def test_build_without_params(self, workflow, params, retrive_build_and_output):
         assert workflow.jenkins.version in ''.join(output)
 
     @pytest.mark.parametrize('params', [{'arg1': 'arg1_value'},
-                                        {'arg1': 'arg1_value', 'delay': 2}],
-                             ids=['without delay', 'with delay'])
+                                        {'arg1': 'arg1_value', 'delay': 2},
+                                        {'arg1': 'arg1_value', 'delay': 2, 'test_file': tempfile.TemporaryFile()}],
```

Comment:
i suppose we need create a new job for testing file parameters instead of reusing existed.